### PR TITLE
Codestyle

### PR DESCRIPTION
**Pull request by sebastian-dev**

### CODE REVIEW

File: GUI/CrashForm.cs [hunk 1]
```diff
@@ -1,91 +1,93 @@
...
-        try {
-          Stream dataStream = request.GetRequestStream();
-          dataStream.Write(byteArray, 0, byteArray.Length);
-          dataStream.Close();
+        private void SendButton_Click(object sender, EventArgs e)
+        {
+            try
+            {
+                Version version = typeof(CrashForm).Assembly.GetName().Version;
+                WebRequest request = WebRequest.Create("http://openhardwaremonitor.org/report.php");
```
Comment (Phyxion) [inline GUI/CrashForm.cs]: I think this send stuff can be deleted in another PR as it's not useful here.

**Comment by Phyxion** [comment]
What do you think about renaming the namespace of OpenHardwareMonitor to LibreHardwareMonitor as well?

**Comment by sebastian-dev** [comment]
> What do you think about renaming the namespace of OpenHardwareMonitor to LibreHardwareMonitor as well?
I would do this when the ui and the lib changes are finished.

File: GUI/SplitContainerAdv.cs [hunk 2]
```diff
@@ -1,123 +1,150 @@
...
-        if (e.KeyData == Keys.Right || e.KeyData == Keys.Down) {
-          SplitterDistance += SplitterIncrement;
-        } else if (e.KeyData == Keys.Left || e.KeyData == Keys.Up) {
-          SplitterDistance -= SplitterIncrement;
+        protected override void OnKeyDown(KeyEventArgs e)
+        {
+            if (!base.IsSplitterFixed)
+            {
+                if (e.KeyData == Keys.Right || e.KeyData == Keys.Down)
+                {
```
Comment (Phyxion) [inline GUI/SplitContainerAdv.cs]: Do you know if there's an editor config to remove braces in a single line if/else? As long as a block is properly indented I think it's unnecessary to have these braces.

**Comment by Phyxion** [comment]
> 
> 
> > What do you think about renaming the namespace of OpenHardwareMonitor to LibreHardwareMonitor as well?
> > I would do this when the ui and the lib changes are finished.

Yeah, sounds good to me. I think in another PR we should also look into moving native stuff to a separate file similar to Kernel32. This would greatly clean up the GUI files.

Style changes look good. There's still some weird things that can still be addressed, I left some comments here and there.

File: GUI/SplitContainerAdv.cs (hunk 2 above)
Comment (sebastian-dev) [reply GUI/SplitContainerAdv.cs]: Sometimes it is an good idea to have braces. (multiline conditions, or long if statements)
this make the code more readable

Comment (Phyxion) [reply GUI/SplitContainerAdv.cs]: In some cases, but in most cases - like this one - it's not an improvement I think. I've been looking into it and you can set csharp_prefer_braces to when_multiline. Then as a suggestion instead of error should be OK.

File: GUI/MainForm.cs [hunk 3]
```diff
@@ -649,13 +649,10 @@ private void Timer_Tick(object sender, EventArgs e)
             _systemTray.Redraw();
             if (_gadget != null)
                 _gadget.Redraw();
-
```
Comment (Phyxion) [inline GUI/MainForm.cs]: Why are all these lines now removed? Is that a cause of the when_multiline? I'd rather have them back after such a statement (same as in the other files).

Comment (sebastian-dev) [reply GUI/MainForm.cs]: i deleted this lines, for this short functions i dont need any spacing lines for reading.

Comment (Phyxion) [reply GUI/MainForm.cs]: After brackets or control flow changes should be a new line for readability. I'll make a couple of changes, rest looks good now. Somehow this PR got broken as GitHub doesn't see this branch anymore so I'll recreate it.
